cg/meta/store/base.py

```diff
@@ -27,12 +78,77 @@ def add_new_analysis(bundle_data, case_obj, status, version_obj):
     return new_analysis
 
 
+def parse_config(data: dict) -> dict:
+    """Parse MIP config file.
+
+    Args:
+        data (dict): raw YAML input from MIP analysis config file
+
+    Returns:
+        dict: parsed data
+    """
+    return {
+        "email": data.get("email"),
+        "case": data["case_id"],
+        "samples": _get_sample_analysis_type(data),
+        "is_dryrun": bool("dry_run_all" in data),
+        "out_dir": data["outdata_dir"],
+        "priority": data["slurm_quality_of_service"],
+        "sampleinfo_path": data["sample_info_file"],
+    }
+
+
+def parse_sampleinfo(data: dict) -> dict:
```

Comment:
```suggestion
def parse_mip_sampleinfo(data: dict) -> dict:
```

Reply:
Another example of things that might belong in a store/mip.py

Reply:
Also resolved in https://github.com/Clinical-Genomics/cg/pull/621/commits/a6295bbeb481265db210bcf6eb8fbe9deaae79dc